ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,22 @@
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A clear and concise description of what actually happen.
+
+**Steps to reproduce the behaviour**
+1. Go to '...'
+2. Run '....'
+...
+4. See error
+
+**Platform (please complete the following information):**
+ - OS: [e.g. Linux 4.16]
```

Comment:
@Koroqe  
Is not contracts development platform agnostic? 
Is there any reason to have this info in each one issue?

Reply:
There could be issues related to truffle, ganache and solc setup on different OS. But, yes, we can exclude it from template